contracts/GSN/IRelayRecipient.sol

```diff
@@ -1,14 +1,32 @@
 pragma solidity ^0.5.0;
 
-/*
- * @dev Interface for a contract that will be called via the GSN from RelayHub.
+/**
+ * @dev Base interface for a contract that will be called via the GSN from {IRelayHub}. Do not use directly, inherit from
+ * {GSNRecipient} instead.
  */
 contract IRelayRecipient {
     /**
-     * @dev Returns the address of the RelayHub instance this recipient interacts with.
+     * @dev Returns the address of the {IRelayHub} instance this recipient interacts with.
      */
     function getHubAddr() public view returns (address);
```

Comment:
Is there a reason that `getHubAddr` is abbreviated rather than `getHubAddress`?
Assume to late to change interface.